arch/xtensa/include/xtensa-asm2-context.h

```diff
@@ -85,12 +85,19 @@
 #define BASE_SAVE_AREA_SIZE_THREADPTR	0
 #endif
 
+#if XCHAL_HAVE_FP && defined(CONFIG_CPU_HAS_FPU) && defined(CONFIG_FPU_SHARING)
+#define BASE_SAVE_AREA_SIZE_FPU		18 * 4
```

Comment:
Nit: Why not put the final value directly here?

Reply:
I find it more readable than plain "72". I think it becomes clearer that an additional 18 x 32bit space on the stack is needed. But I can change it if you disagree, no problem at all.

Reply:
Please put parentheses around this.